website/src/components/MiniBlog/styles.ts

```diff
@@ -6,44 +6,55 @@ const useStyles = makeStyles((theme: Theme) => ({
     textAlign: "center",
     padding: theme.spacing(5, 5, 0),
   },
-  tabLayout: {
-    orientation: "vertical",
+  tabsWrapper: {
+    justifyContent: 'center'
   },
   tabs: {
-    maxWidth: "auto",
-    margin: `${theme.spacing(1)}px auto`,
+    maxWidth: "100%",
     padding: theme.spacing(0.3),
     backgroundColor: "transparent",
     flexGrow: 1,
     fontWeight: 700,
     boxShadow: "2px 0px 33px 5px rgba(70, 68, 151, 0)",
   },
+  tabRoot: {
+    justifyContent: "center",
+    '& .Mui-selected': {
+      color: theme.palette.text.hint,
+      background: theme.palette.background.paper,
+      boxShadow: '2px 5px 31px 4px rgba(70, 68, 151, 0.07)',
```

Comment:
should we try using color from palette?

Reply:
The colors used in box shadows are unique. Not used elsewhere on the website. Currently, for all box-shadow values, we have directly included the rgba value. We can refractor all of them separately if required

Reply:
it will be good to have all the colors in one place.

Reply:
Yes, will refractor it separately in another PR along with all other box-shadow colors